.circleci/integration-tests/master_dag.py

```diff
@@ -18,7 +18,8 @@
 SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "#provider-alert")
 SLACK_WEBHOOK_CONN = os.getenv("SLACK_WEBHOOK_CONN", "http_slack")
 SLACK_USERNAME = os.getenv("SLACK_USERNAME", "airflow_app")
-MASTER_DAG_SCHEDULE = os.getenv("MASTER_DAG_SCHEDULE", None)
+MASTER_DAG_SCHEDULE = os.getenv("MASTER_DAG_SCHEDULE", "0 0 * * *")
```

Comment:
A change was made recently in the GitHub actions deploy image job which triggers the master DAG after deploying. So, we do not need to have a scheduled run. Also, we do not want to run the master DAG at this schedule as @Lee-W starts his day during the run of this DAG and it creates a hurdle for him. So keeping set it to None is what we need at least for the alerts that we have in #provider-alert channel.